docs/howto/manage-charms.rst

```diff
@@ -65,6 +66,34 @@ in, a charm name different from the name of the root directory, etc.
 
     See more: :ref:`manage-extensions`
 
+The profiles also all provide recommended tooling and configuration for
+developing charms:
+
+- Run ``tox -e format`` to automatically format the charm and test code.
+- Run ``tox -e lint`` to lint the charm.
+- Run ``tox -e unit`` to run the charm unit tests.
+- Run ``tox -e static`` to run static type checking.
+- Run ``tox`` to run all of the above.
+- Run ``tox -e integration`` to run the charm integration tests.
+
+.. admonition:: Best Practice
+    :class: hint
+
+    The quality assurance pipeline of a charm should be automated using a
+    continuous integration (CI) system.
+
```

Comment:
FWIW I do not think this is redundant -- I can definitely see it being useful to some of our intended target audience.